app/controllers/jams_controller.rb

```diff
@@ -10,6 +10,14 @@ def create
 
     if jam.save
       jam.activities.create!(user: current_user)
+      room.users.reject{|user| user == current_user }.each do |user|
+        NotificationService.notify(
+          subject: jam,
+          user: user,
+          event: :jam_created
+        )
+      end
+
```

Comment:
that feels like a lot of logic for a controller already. I'd add a method in `NotificationService` for now (could be more generic in the future) like so: `NotificationService.notify_on_jam_creation(jam, current_user)`

Reply:
(potentially, `jam.activities.create!(user: current_user)` could be moved into that method too)